src/SFA.DAS.EmployerAccounts/Commands/AuditCommand/CreateAuditCommandValidator.cs

```diff
@@ -24,7 +24,10 @@ public ValidationResult Validate(CreateAuditCommand item)
 
             if (item.EasAuditMessage.ChangedProperties==null || !item.EasAuditMessage.ChangedProperties.Any())
             {
-                validationResult.AddError(nameof(item.EasAuditMessage.ChangedProperties));
+                if (item.EasAuditMessage.Category == null || !item.EasAuditMessage.Category.Equals("VIEW", StringComparison.InvariantCultureIgnoreCase))
```

Comment:
thought we had gotten rid of the requirement to audit when the support user view something?

Reply:
No it's acceptance criteria 1 in the story.

Reply:
ok